### PR TITLE
Enhancement/4697 isgatheringdata for all widgets

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue:

- #4697 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserCountGraph.js [hunk 1]
```diff
@@ -47,7 +47,9 @@ const MOBILE_TO_DESKOP_MEDIA_QUERY =
 	'(min-width: 451px) and (max-width: 1280px';
 const X_LARGE_AND_ABOVE_MEDIA_QUERY = '(min-width: 1281px)';
 
-export default function UserCountGraph( { loaded, error, report } ) {
+export default function UserCountGraph( props ) {
+	const { loaded, error, report, gatheringData } = props;
```
Comment (asvinb) [inline assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserCountGraph.js:51]: @eugene-manuilov Does that change make any difference? 🤔 Or we could have just added `gatheringData` to the list of destructured props in the function parameter?

**Review by asvinb — COMMENTED** [review]
@eugene-manuilov The PR LGTM, but I left a small comment. Can you check it out?

File: assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserCountGraph.js (hunk 1 above)
Comment (asvinb) [reply assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserCountGraph.js:51]: Gotcha! Thanks @eugene-manuilov

**Review by tofumatt — APPROVED** [review]
Looks great, tested locally and works as-expected 👍🏻

**Pull request merged**